.travis.yml

```diff
@@ -35,8 +36,10 @@ env:
     - DOCKER_VERSION=1.10.3-0~trusty
     - DOCKER_COMPOSE_VERSION=1.6.2
   matrix:
+    - SUITE=pep8
```

Comment:
can we also add another variable like `NEED_SETUP=true` in the matrix so that for the `PEP8` Build we don't do the whole build step and just pull python-base on `docker-compose run`.